### PR TITLE
Github achievements in Swahili : Updates

### PR DESCRIPTION
**Pull request by the1Riddle**
Made changes requested from the previous [PR](https://github.com/4xmen/Get-Github-Achievements/pull/145#issuecomment-1968534665) on creating the buttons.

### CODE REVIEW

File: README.sw.md [hunk 1]
```diff
@@ -146,13 +146,13 @@
 
 #### - Ikiwa unahitaji usaidizi zaidi, bofya kitufe cha `Jinsi ya Kupata` ili kupata mafunzo ya hatua kwa hatua kuhusu jinsi ya kupata mafanikio haya.
 
-<div align="center"><a href="assets/steps/README.starstruck.md"><img src="assets/img/btn.png" alt=""></a></div>
+<div align="center"><a href="assets-sw/steps/README.starstruck-sw.md"><img src="assets/img/btn.png" alt=""></a></div>
```
Comment (the-1Riddle) [inline README.sw.md:149]: this would link to a broken link it should be assets/steps-sw instead

**Comment by the1Riddle** [comment]
hello @YasinDehfuli done with the implementation requested.

**Comment by YasinDehfuli** [comment]
Thanks Dear @the1Riddle & @the-1Riddle. 

Everything is done. 

But please Add Swahili button to other translated languages. (Arabic, Japanese, Dutch &…)

I'm happy to see you as our contributor.

With Respect @YasinDehfuli

**Comment by the1Riddle** [comment]
on it :rocket:

**Pull request merged**